app/src/main/java/io/awaken/ui/connections/ConnectionViewHolder.kt

```diff
@@ -66,19 +68,29 @@ class ConnectionViewHolder(
             val builder = AlertDialog.Builder(itemView.context, R.style.AlertDialog)
             builder.setMessage("Are you sure you want to delete " + nickname.text.toString() + "?")
             builder.setNegativeButton(android.R.string.cancel, null)
-            builder.setPositiveButton(android.R.string.ok) { dialog, which ->
+            builder.setPositiveButton(android.R.string.ok) { _, _ ->
                 var message: String
+                var connection: Connection? = null
                 try {
+                    connection = databaseHelper.getConnection(connectionId)
                     databaseHelper.deleteConnection(connectionId)
                     message = "Successfully deleted " + nickname.text.toString()
                     refreshListener.invoke()
-                }
-                catch (e: Exception) {
+                } catch (e: Exception) {
                     message = "Failed to delete " + nickname.text.toString()
                     Log.e("failure", e.message)
                 }
 
-                Toast.makeText(itemView.context, message, Toast.LENGTH_LONG).show()
+                Snackbar.make(itemView, message, Snackbar.LENGTH_LONG)
```

Comment:
Do we want to show this if the connection wasn't actually deleted? What if `connection` is null?